text/0000-rfc-stages.md

```diff
@@ -0,0 +1,189 @@
+
+1. Community feedback that it was difficult to follow along with the progression of Ember Octane's features
+2. Challenges uncovered in the existing RFC Tracking process
+3. The long-standing, but stagnant goal of providing a public-facing status board
+4. Confusion about what it means for an RFC to be merged. There is a common myth that a merged RFC indicates that a feature will be in the next release, when instead it is a green flag for implementation work to begin.
+
+To understand the problem, it is helpful to think of the informal stages a merged RFC already can progress through: We have RFCs for which we have needed to indicate a change in direction ([MU](https://emberjs.github.io/rfcs/0143-module-unification.html)), RFCs that have needed to be clarified/replaced ([Named Blocks](https://emberjs.github.io/rfcs/0226-named-blocks.html)), RFCs waiting to be implemented, and RFCs whose features have been included in a stable release. Each of these scenarios was handled in a thoughtful manner, but they were difficult for users to follow along with.
+
+This proposal aims to build on the success of RFC 300, [RFC (Request for Comments) Process Update]([https://github.com/emberjs/rfcs/blob/master/text/0300-rfc-process-update.md](https://github.com/emberjs/rfcs/blob/master/text/0300-rfc-process-update.md)).
+
+## Detailed design
+
+### Stages:
+
+A successful RFC can move through the following stages:
+
+| Stage| Description| Criteria to advance to next step |
+| -----| -----------|----------------------------------| 
+| Proposed | A pull request made to the RFC repository which is open for public comment and core team evaluation. | "The RFC has been in Final Comment Period, and the relevant core team has consensus to merge it, thus accepting the proposal." |
+| Accepted | "The work to integrate the proposal into Ember's codebases or materials may begin. An ""Accepted"" RFC is not necessary in-progress." | "The technical implementation is complete according to plan outlined in the RFC, and is in harmony with any changes in Ember that have occurred since the RFC was first written." |
+| Prepared for Release | "The relevant core team reviews the work to determine when it can be included in a stable release. At this stage, features or deprecations may be available for use behind a feature flag, in beta, etc. This is also a time to queue up learning materials, for example." | "The relevant core teams coordinate to include the work in a stable release, and/or publish the materials. If there are any truly critical deviations from the original RFC, they are briefly noted at the top of the RFC." |
+| Stable | "The work is in a stable release/is published. If it is codebase-related work, it adheres to SemVer from this point onward." | "If it is a codebase change, it is well documented and has clear migration paths. It is consistent with Ember's mental models." |
+| Complete | The feature/resource is recommended for general use. | n/a |
```

Comment:
I feel like these stages do a good job of capturing status after RFCs are merged, but the "Proposed" step is what needs to be expanded/exploded. As an RFC author, that part is completely opaque and really doesn't motivate me to write more RFCs.

If these were design documents / RFCs for a product, there would be some SLA or some kind of rubric to determine how to move forward with a document.

Another thing to think about is expirations. I think it's totally reasonable to expire an RFC and close it out if it doesn't receive either consensus or updates for N months (or whatever the criteria is). It's incredibly frustrating to see RFCs stay open forever. I imagine that this would reduce maintainer burden as well. PRs and Issues don't go anywhere, so it would be fine to open them back up again or resubmit some other way.

Reply:
I think you are correct in that this RFC is not exactly intended to address those particular problems with RFCs. I do think the `Exploring` stage will help for a subset -- the core teams can quickly indicate that an RFC deserves further effort. 

Recently, at least the framework team, we have made an effort to ensure team notes have been published much more frequently. That said, when an RFC is discussed, we nearly always comment on that RFC, either with requested changes or any action that was decided.